### PR TITLE
update theia prod ingress

### PR DESCRIPTION
**Pull request by yuenmichelle1**
Currently putting in Draft to double check all theia maintenance tasks are done

### CODE REVIEW

**Review by camallen — APPROVED** [review]
LGTM - if you are happy that Theia prod pod is good to go then :shipit:

**Comment by yuenmichelle1** [comment]
@zwolf  most maintenance tasks are done, but let me know if i missed anything via DM. I'll wait for your approval to merge. no rush on this though :)

**Review by zwolf — APPROVED** [review]
LGTM ¯\_(ツ)_/¯

**Pull request merged**
